gensim/test/test_wikicorpus.py

```diff
@@ -30,10 +30,11 @@ def setUp(self):
         wc = WikiCorpus(datapath(FILENAME))
 
     def test_get_texts_returns_generator_of_lists(self):
+
```

Comment:
It was suggested by Lev to leave a blank line to pass the Travis build. I am not sure why.